### ISSUE TITLE
Unable to play Apple Lossless .m4a files

### ISSUE THREAD

**Issue by user-julian** [karimknaebel/phonograph-issue-tracker#184]
Apple Lossless .m4a files look like they are playing but don't progress and are silent. This means that whenever one comes up on the play queue all music stops. If Phonograph can't play Apple Lossless files it should at least skip them, or notify the user. Still, best music player for Android.

![2017-01-16 20 40 10](https://cloud.githubusercontent.com/assets/17108837/21998326/23d49ec0-dc02-11e6-8aab-b8bdacbef49b.png)
![2017-01-16 20 38 54](https://cloud.githubusercontent.com/assets/17108837/21998303/0100ac22-dc02-11e6-9f6c-8885f765e8d6.png)

Device info:
---
<table>
<tr><td>App version</td><td>0.13.5</td></tr>
<tr><td>App version code</td><td>124</td></tr>
<tr><td>Android build version</td><td>3407729</td></tr>
<tr><td>Android release version</td><td>7.0</td></tr>
<tr><td>Android SDK version</td><td>24</td></tr>
<tr><td>Android build ID</td><td>NBD91T</td></tr>
<tr><td>Device brand</td><td>google</td></tr>
<tr><td>Device manufacturer</td><td>LGE</td></tr>
<tr><td>Device name</td><td>bullhead</td></tr>
<tr><td>Device model</td><td>Nexus 5X</td></tr>
<tr><td>Device product name</td><td>bullhead</td></tr>
<tr><td>Device hardware name</td><td>bullhead</td></tr>
<tr><td>ABIs</td><td>[arm64-v8a, armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (32bit)</td><td>[armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (64bit)</td><td>[arm64-v8a]</td></tr>
</table>

**Comment by arkon** [comment]
Agreed, we should skip/notify if we come across files that aren't supported.
